guides/v2.2/javascript-dev-guide/widgets/widget-multiselect.md

```diff
@@ -124,6 +125,112 @@ The search field update delay.
 
 **Default value**: `0`
 
+### `nextPageUrl`
+
+The url of the controller or API which should return the JSON response like the following:
+
+```json
+{
+  "success": true,
+  "errorMessage": "",
+  "result": [
+    {
+      "value": "1",
+      "label": "Multi-select item label #1"
+    },
+    {
+      "value": "2",
+      "label": "Multi-select item label #2"
+    }
+  ]
+}
+```
+
+The example of the controller: [`<Magento_Tax_module_dir>/Controller/Adminhtml/Rule/AjaxLoadRates.php`][] 
+
+**Type**: String
+
+**Default value**: `undefined`
+
+## Methods
+
+The Multiselect methods are the following:
+
+- [getCurrentPage()](#getcurrentpage)
+- [setCurrentPage()](#setcurrentpage)
+- [isOptionsLoaded()](#isoptionsloaded)
+- [clearMultiselectOptions()](#clearmultiselectoptions)
+- [appendOptions()](#appendoptions)
+- [loadOptions()](#loadoptions)
+- [getSearchCriteria()](#getsearchcriteria)
+
+### `getCurrentPage()`
+
+The method returns the number of the current page.
+
+```javascript
+$('#multiselect').multiselect2('getCurrentPage');
+```
+
+### `setCurrentPage(page)`
+
+The method configures the current page number.
+
+```javascript
+$('#multiselect').multiselect2('setCurrentPage', 2);
+```
+
+### `isOptionsLoaded(options)`
+
+The method checks if all multi-select items are already loaded.
+
+```javascript
+$('#multiselect').multiselect2('isOptionsLoaded', [{"value": "2", "label": "Label #2"}]);
+```
+
+This method returns `true` if multiple selection items are loaded, and it returns `false` if they are not loaded. 
+
+### `clearMultiselectOptions()`
+
+The method clears all multi-select items.
```

Comment:
Could you clarify this point, please? The "clears" means deselect items or remove them? 
Thank you

Reply:
The `clearMultiselectOptions()` method removes all multi-select items. The description of this method already adjusted. Thank you!